config/vmdb.tmpl.yml

```diff
@@ -238,6 +238,9 @@ smtp:
 snapshots:
   :create_free_percent: 100
   :remove_free_percent: 100
+ssl:
+  :ssl_ca_file:
+  :ssl_ca_path:
```

Comment:
These should be grouped under the [openstack event catcher configs](https://github.com/Ladas/manageiq/blob/af91bd49d6ca243eaa8c6f2122e216d574665929/config/vmdb.tmpl.yml#L302).  Putting it at this level implies that this is the *only* CA File for all of ManageIQ.  But, I suspect that we're going to need to support different CA files for different services ... possibly.

Ultimately, we'll likely need to support configuring this at the Provider level.  Because even grouping under OpenStack implies that *all* OpenStack environments will use the same root cert.